riscv/src/continuations/bootloader.rs

```diff
@@ -24,10 +24,6 @@ pub const BOOTLOADER_SPECIFIC_INSTRUCTION_NAMES: [&str; 2] =
 
 pub fn bootloader_preamble() -> String {
     let mut preamble = r#"
-    // ============== extra rules on memory when compiled with continuations =======================
-    // The first operation can't be a read (because we expect this page to be paged-in)
-    m_change * m_is_read' = 0;
```

Comment:
This was actually a soundness bug :/

It only says that the next operation after an address change, can't be a read. But the constraints allow for it to be neither a read or a write. Then, the row after can be a read.

With the constraints now, this is replaced with `m_change * (1 - m_is_bootloader_write') = 0`.